server/integration/infinispan/pom.xml

```diff
@@ -138,6 +143,35 @@
                 </executions>
             </plugin>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>serialize_component_metadata</id>
+                        <phase>process-classes</phase>
+                        <goals>
+                            <goal>java</goal>
```

Comment:
If it were a sub-module of infinispan parent, simply adding 
   <properties>
         <module.skipComponentMetaDataProcessing>false</module.skipComponentMetaDataProcessing>
   </properties>
would do the trick.

Unfortunately, the "parent hierarchy" for this module is:
infinispan-server-versions <- jboss-parent